githubgen/main.go

```diff
@@ -169,7 +169,7 @@ func newIssueTemplatesGenerator(trimSuffixes string) *issueTemplatesGenerator {
 }
 
 func newCodeownersGenerator(skipGithubCheck *bool) *codeownersGenerator {
-	return &codeownersGenerator{skipGithub: *skipGithubCheck, getGitHubMembers: GetGithubMembers}
+	return &codeownersGenerator{skipGithub: *skipGithubCheck, getGitHubMembers: getGithubMembers, getFile: getFile, setFile: setFile}
```

Comment:
Maybe you can introduce a NewCodeownersGenerator() func which sets the `setFile` and `getFile` by default (maybe also the other parameters, if they are not from user input)

Reply:
doesn't my factory function do exactly that?

Reply:
oh sorry my bad, read the code wrongly, all good here